src/DynamoCoreWpf/Views/Menu/PreferencesView.xaml

```diff
@@ -159,135 +159,205 @@
-                                </Grid.RowDefinitions>
-                                <Label Grid.Row="0"
-                                       Content="{x:Static p:Resources.DynamoViewSettingMenuNumberFormat}" 
-                                       Margin="-3,0,10,0"
-                                       FontWeight="Bold"
-                                       Foreground="{StaticResource PreferencesWindowFontColor}"/>
-                                <ComboBox Name="numberFormatCmb"
-                                          Grid.Row="1"                                         
-                                          SelectedItem="{Binding Path=SelectedNumberFormat, Converter={StaticResource NumberFormatConverter}}"
-                                          ItemsSource="{Binding NumberFormatList}"
-                                          Style="{StaticResource NoBordersComboBoxStyle}"
-                                          Width="213">
-                                </ComboBox>
+                                <!--This Grid contains controls for selecting Number Format options-->
+                                <Grid Grid.Row="4" Margin="0,0,0,20">
+                                    <Grid.RowDefinitions>
+                                        <RowDefinition Height="Auto" />
+                                        <RowDefinition Height="Auto" />
+                                    </Grid.RowDefinitions>
+                                    <Label Grid.Row="0"
+                                        Content="{x:Static p:Resources.DynamoViewSettingMenuNumberFormat}" 
+                                        Margin="-3,0,10,0"
+                                        FontWeight="Bold"
+                                        Foreground="{StaticResource PreferencesWindowFontColor}">
+                                    </Label>
+                                    <ComboBox Name="numberFormatCmb"
+                                            Grid.Row="1" 
+                                            HorizontalAlignment="Left"
+                                            SelectedItem="{Binding Path=SelectedNumberFormat, Converter={StaticResource NumberFormatConverter}}"
+                                            ItemsSource="{Binding NumberFormatList}"
+                                            Style="{StaticResource NoBordersComboBoxStyle}"
+                                            Width="213">
+                                      </ComboBox>
+                                </Grid>
+
+                                <Grid Grid.Row="5">
+                                    <!--Geometry Scaling Expander-->
+                                    <Expander x:Name="Scale"
+                                      DataContext="{Binding OptionsGeometryScale}"
+                                      Style="{StaticResource MenuExpanderStyle}" 
+                                      Expanded="Expander_Expanded"
```

Comment:
I think @RobertGlobant20 touched this in a different PR so there is probably some collaboration here :)

Reply:
Now that PR from @RobertGlobant20 is merged, there are anticipated conflicts, would you address that? Thanks!